dashboard/app/models/stage.rb

```diff
@@ -94,8 +94,14 @@ def localized_category
     end
   end
 
+  def localized_lesson_plan
+    if script.curriculum_path?
+      script.curriculum_path.gsub('{LESSON}', relative_position.to_s).gsub('{LOCALE}', '')
+    end
+  end
+
   def lesson_plan_html_url
-    "#{lesson_plan_base_url}/Teacher"
+    localized_lesson_plan || "#{lesson_plan_base_url}/Teacher"
   end
 
   def lesson_plan_pdf_url
```

Comment:
should this be updated as well?

Reply:
Only the old format of lesson plans have PDF links.  AFAIK they're only used on unplugged levels, e.g. https://studio.code.org/s/course2/stage/1/puzzle/1.